src/uiSelectChoicesDirective.js

```diff
@@ -68,7 +66,7 @@ uis.directive('uiSelectChoices',
         scope.$watch('$select.search', function(newValue) {
           if(newValue && !$select.open && $select.multiple) $select.activate(false, true);
           $select.activeIndex = $select.tagging.isActivated ? -1 : 0;
-          if (!attrs.minimumInputLength || $select.search.length >= attrs.minimumInputLength) {
+          if ((!attrs.minimumInputLength || $select.search.length >= attrs.minimumInputLength) && $select.open) {
```

Comment:
I changed this as  this will prevent the initial request of the refresh function on initializing.
Yes reverting will fix the tests.